### ISSUE TITLE
FIX: Broken Links and Redirects

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/lecture-python.myst#261]
@HumphreyYang would you mind working your way through these and setup a PR when you have time

Not sure how to solve the acemoglu link issue. 

```
ols.md:89: [broken] http://economics.mit.edu/faculty/acemoglu/data/ajr2001: 404 Client Error: Not Found for url: https://economics.mit.edu/faculty/acemoglu/data/ajr2001
ols.md:50: [broken] http://economics.mit.edu/files/4123: 404 Client Error: Not Found for url: https://economics.mit.edu/files/4123
linear_algebra.md:1158: [redirected permanently] http://docs.scipy.org/doc/scipy/reference/linalg.html to https://docs.scipy.org/doc/scipy/reference/linalg.html
intro.md:14: [redirected permanently] http://johnstachurski.net/ to https://johnstachurski.net/
coleman_policy_iter.md:132: [redirected permanently] http://johnstachurski.net/edtc.html to https://johnstachurski.net/edtc.html
two_auctions.md:654: [redirected permanently] http://faculty.smu.edu/tsalmon/auctions.pdf to https://s2.smu.edu/tsalmon/
pandas_panel.md:499: [redirected permanently] http://ec.europa.eu/eurostat/data/database to https://ec.europa.eu/eurostat/data/database
aiyagari.md:190: [redirected permanently] http://quantecon.org/quantecon-py to https://quantecon.org/quantecon-py/
aiyagari.md:73: [redirected with Found] http://nbviewer.jupyter.org/github/QuantEcon/QuantEcon.notebooks/blob/master/aiyagari_continuous_time.ipynb to https://nbviewer.org/github/QuantEcon/QuantEcon.notebooks/blob/master/aiyagari_continuous_time.ipynb
lake_model.md:634: [redirected permanently] http://www.cdc.gov/nchs/fastats/births.htm to https://www.cdc.gov/nchs/fastats/births.htm
intro.md:1: [redirected permanently] http://rst-python.quantecon.org to https://d6mtww49nma8j.cloudfront.net/
lake_model.md:634: [redirected permanently] http://www.cdc.gov/nchs/fastats/deaths.htm to https://www.cdc.gov/nchs/fastats/deaths.htm
mle.md:38: [redirected permanently] http://pubs.aeaweb.org/doi/pdfplus/10.1257/aer.p20161068 to https://pubs.aeaweb.org/doi/pdfplus/10.1257/aer.p20161068
linear_algebra.md:536: [redirected permanently] http://www.mathsisfun.com/algebra/matrix-multiplying.html to https://www.mathsisfun.com/algebra/matrix-multiplying.html
mle.md:768: [redirected permanently] http://www.statsmodels.org/dev/generated/statsmodels.discrete.discrete_model.Probit.html to https://www.statsmodels.org/dev/generated/statsmodels.discrete.discrete_model.Probit.html
mle.md:773: [redirected permanently] http://www.statsmodels.org/dev/examples/notebooks/generated/generic_mle.html to https://www.statsmodels.org/dev/examples/notebooks/generated/generic_mle.html
mle.md:768: [redirected permanently] http://www.statsmodels.org/dev/generated/statsmodels.discrete.discrete_model.Logit.html to https://www.statsmodels.org/dev/generated/statsmodels.discrete.discrete_model.Logit.html
ols.md:555: [redirected permanently] http://www.statsmodels.org/dev/example_formulas.html to https://www.statsmodels.org/dev/example_formulas.html
linear_algebra.md:1160: [redirected permanently] http://www.johnstachurski.net/emet.html to https://johnstachurski.net/emet.html
status.md:19: [redirected permanently] https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources to https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
pandas_panel.md:490: [redirected with Found] http://xarray.pydata.org/en/stable/ to https://docs.xarray.dev/en/stable/
troubleshooting.md:40: [redirected permanently] https://www.anaconda.com/keeping-anaconda-date/ to https://www.anaconda.com/blog/keeping-anaconda-date
```

**Comment by HumphreyYang** [comment]
> Not sure how to solve the acemoglu link issue.

Hi @mmcky, I found MIT is still constructing the [new website](https://economics.mit.edu/people/faculty/daron-acemoglu/data-archive) for the data archive for Professor Daron Acemoglu. I can find a copy of the old website on the Wayback Machine. From the terms and conditions, I did not see anything preventing us from using the resource on the website. What is your opinion on including the Wayback machine archive link as a temporary solution?
